Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonRecentDoc.cs

```diff
@@ -73,10 +73,10 @@ public string Text
                 // We never allow an empty text value
                 if (string.IsNullOrEmpty(value))
                 {
-                    value = "Recent Document";
+                    value = @"Recent Document";
                 }
 
-                if (value != _text)
+                if (value != null && value != _text)
```

Comment:
`value` has already been checked for `null` on line 74

Reply:
Removed check